### ISSUE TITLE
All files diffs

### ISSUE THREAD

**Issue by larshp** [larshp/abapGit#2111]
after #2096 @sbcgua 

![image](https://user-images.githubusercontent.com/5888506/48657396-82e25180-ea30-11e8-8c16-4aaabe18cfa1.png)

**Comment by sbcgua** [comment]
Yeah, vigilance dropped in the final PR commit and the bug happened. Thanks for the fix :)
